core/controlplane/config/templates/cloud-config-controller

```diff
@@ -3323,7 +3323,19 @@ write_files:
           {{end}}
           resources:
             requests:
-              cpu: 200m
+              cpu: {{ if .Kubernetes.ControllerManager.ComputeResources.Requests.Cpu }}{{ .Kubernetes.ControllerManager.ComputeResources.Requests.Cpu }}{{ else }}200m{{ end }}
```

Comment:
Nit: We'd better define the default of `200m` in go and `cluster.yaml` template, similarly to other settings.